cleanslate/doma/static/css/base.css

```diff
@@ -0,0 +1,17 @@
+.navbar-light .navbar-nav .nav-link {

```

Comment:
Think the items between the navbar might need a bit more space.

Reply:
I'll give it a 15px padding either side